src/ui/verificationSymbolStatusView.ts

```diff
@@ -62,17 +71,35 @@ export default class VerificationSymbolStatusView {
     return listener;
   }
 
-  private constructor(
+  public constructor(
     private readonly context: ExtensionContext,
     private readonly languageClient: DafnyLanguageClient,
     private readonly compilationStatusView: CompilationStatusView) {
     this.controller = this.createController();
+
+    workspace.onDidChangeTextDocument(e => {
+      const uriString = e.document.uri.toString();
+      this.updatesPerFile.delete(uriString);
+      this.updateListenersPerFile.delete(uriString);
```

Comment:
Will these updates be automatically immediately regenerated? Couldn't you delete them only when the language server returns new ones to override?

Reply:
This comment as well.